src/scripts/createGame.ts

```diff
@@ -4,7 +4,13 @@ import Word, { WordT } from '../models/Word';
 import { generateLetters } from '../utils/generateLetters';
 
 const calcLevels = (words: WordT[], panagrams: WordT[]) => {
-  const totalWordLength = words.reduce((acc, curr) => acc + curr.word.length, 0);
+  /*  const totalWordLength = words.reduce((acc, curr) => acc + curr.word.length, 0); */
+  const shortWords = words.filter((obj) => obj.word.length === 4);
```

Comment:
Nice, but can be improved by not iterating through words twice in a row. Would go for a good ol for of loop here.